src/process-test-results.js

```diff
@@ -11,7 +11,8 @@ function processTestResults(lastResults, flakyResults) {
     if (outdatedResult) {
       outdatedResult.testResults.forEach((result, idx) => {
         if (result.status === 'failed') {
-          flaky.testResults[idx].title += ' FLAKY';
+          // Conflicts with flakyNumRetries=Infinity
+          // flaky.testResults[idx].title += ' FLAKY';
```

Comment:
looks like you are using a different strategy to map the failing tests, which is much better 👍. Did you forget to remove these code here? If you want to keep this, maybe just comment the whole block out.

Reply:
Ya this title mutation isn't entirely useful right now, and I was pretty sure it'd conflict with the failing tests map comparison.  Ideally, we'd create a new column in the jest-junit report (and ultimately jenkins UI) to monitor which are flaky and how many times they were flaky.